pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LLCRealtimeClusterIntegrationTest.java

```diff
@@ -63,6 +63,11 @@
   private final boolean _enableLeadControllerResource = RANDOM.nextBoolean();
   private final long _startTime = System.currentTimeMillis();
 
+  @Override
+  protected boolean injectTombstones() {
+    return true;
```

Comment:
Set this to false to make the test pass at eb6800da96a44e6f5125097cc99a368c2f8f8847